core/src/code.rs

```diff
@@ -68,6 +70,72 @@ fn get_exports(
     Ok(exports)
 }
 
+fn get_stack_end_init_code(module: &Module) -> Option<&[Instruction]> {
+    let Some(section) = module.export_section() else {
+        return None;
+    };
```

Comment:
You can use `?` with `Option`.
```suggestion
    let section = module.export_section()?;
```